src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,38 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            return (
+              <li
+                key={tab.id}
+                className={cn({ 'is-active': activeTabId === tab.id })}
+                data-cy="Tab"
+              >
+                <a
+                  href={`#${tab.id}`}
+                  data-cy="TabLink"
+                  id={tab.id}
+                  onClick={event => {
+                    if (activeTabId !== tab.id) {
+                      onTabSelected(event.target.id);
```

Comment:
instead of  `event.target.id` you can use `onClick={() => onTabSelected(tab.id)}`